tests/test_nitcorn.nit

```diff
@@ -30,23 +30,49 @@ class MyAction
 	redef fun answer(request, turi)
 	do
 		var rep = new HttpResponse(200)
-		rep.body = """
+		var body = """
```

Comment:
As pointed out in #2703, this should be a Buffer rather than a String here

Reply:
What's the point ?

Reply:
Because strings are immutable, each concatenation will result in a new allocation and two `memmove`. A buffer will over-allocate and fill data in-place. It is likely more efficient to allocate a Buffer from the superstring here, and chain `append` statements after that before returning it as a String (maybe not necessary since `Buffer` is writable).

Reply:
Oh. That's the test.
Nevermind then

Reply:
@R4PaSs this is just a basic test that need to respond a string (bytes and other things are tested after). Creating the string with + or with a buffer does not really change something. We kept the + because it is what was used before.

Reply:
This is just a test case. Since body is a Writable, nitcorn doesn't care whether users use a String or a Buffer.